### ISSUE TITLE
Dji Dock and Payload SDK

### ISSUE THREAD

**Issue by abeldixon** [dji-sdk/Payload-SDK#133]
I would like to understand more about the behaviour of the Payload SDK when it used along with the DJI Dock, I have tried giving go to location to DJI Dock and then mission from the Payload SDK, but the mission pause is not working from PSDK, I am running the setup in NVIDIA Jetson.

**Comment by dji-dev** [comment]
Agent comment from shiyao.liao in <a href='https://djisdksupport.zendesk.com/agent/tickets/94729'>Zendesk ticket #94729</a>:<br><br>Dear Developer,

Hello, thank you for contacting DJI.

I'm sorry to bring you a bad experience. I would like to confirm with you that what you mentioned does not work. Is there any error reported in PSDK?

Thank you for your support of DJI products.

Best Regards,
DJI Dajiang innovation SDK technical support
<br><p><em><small>°°°</small></em></p>

**Comment by abeldixon** [comment]
![image](https://github.com/dji-sdk/Payload-SDK/assets/145333241/87266ec5-849f-4f98-a9bc-6022fcd1bd36)

I am getting the following error is there any possible resolution for this ?, it seems the psdk is unable to get the current state of the drone.
When I gave mission from PSDK I am able to pause from the Cloud API but not able to resume. (319044 : Task completed,Unable to resume), and Pausing/Resuming/Stopping from the PSDK is responding with the cmd retry failed when the mission is given from the hover state after take off from the dock.

**Comment by dji-dev** [comment]
Agent comment from shiyao.liao in <a href='https://djisdksupport.zendesk.com/agent/tickets/94729'>Zendesk ticket #94729</a>:<br><br>Dear Developer,

Hello, thank you for contacting DJI.

I saw that the error code…000E1 was reported in the error log. This means that the operation has timed out, which is usually caused by the link. At this time, you can check whether your Uart and BULK links are normal.

Thank you for your support of DJI products.

Best Regards,
DJI Dajiang innovation SDK technical support
<br><p><em><small>°°°</small></em></p>

**Comment by abeldixon** [comment]
Yes, the UART connection is still active because it's launching the sample application and establishing the connection with drone with user option selection ?
The mission execution and manual controls are working fine but the pause/resume is not working.

**Comment by dji-dev** [comment]
Agent comment from shiyao.liao in <a href='https://djisdksupport.zendesk.com/agent/tickets/94729'>Zendesk ticket #94729</a>:<br><br>Dear Developer,

Hello, thank you for contacting DJI.

Could you please send us the complete PSDK operation log? Let's take a look at the communication process in PSDK.

Thank you for your support of DJI products.

Best Regards,
DJI Dajiang innovation SDK technical support
<br><p><em><small>°°°</small></em></p>
